webapp/channels/src/components/advanced_text_editor/use_orientation_handler.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useCallback, useEffect, useRef} from 'react';
+
+import type TextboxClass from 'components/textbox/textbox';
+
+import * as UserAgent from 'utils/user_agent';
+
+const useOrientationHandler = (
+    textboxRef: React.RefObject<TextboxClass>,
+    postId: string,
+) => {
+    const lastOrientation = useRef('');
+
+    const onOrientationChange = useCallback(() => {
+        if (!UserAgent.isIosWeb()) {
```

Comment:
This only does something on iOS, so instead of checking the user agent here, could we check it in the `useEffect` so that we never even attach a listener on other platforms?

Reply:
As mentioned below, we're not changing this for now to minimize the additional changes here